flight/constants.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from enum import Enum
+import numpy as np
```

Comment:
Since this is only being used very few times, lets just use the full name here
```suggestion
import numpy
```